File: 1-10/1.answer.c
#include <stdio.h>

int main() {
	int answer = 0;
	short i;

	for (i = 0; i<1000;i++){
		if(!(i % 3)|| !(i % 5)){
			answer += i;
		}
	}
	printf("Answer = %i", answer);
	return 0;
}

File: 1-10/1.answer.go
package main

import "fmt"

func main() {
    var answer int

	for i := 0; i < 1000; i++ {
		if (i % 3 == 0) || (i % 5 == 0) {
			answer += i;
		}
	}

    fmt.Println("Answer = ", answer);
}

